lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -176,11 +176,10 @@ class DetailsRenderer {
    * @return {Element}
    */
   _renderThumbnail(details) {
-    const element = this._dom.createElement('img', 'lh-thumbnail');
+    const element = this._dom.createElement('div', 'lh-thumbnail');
     const strValue = details;
-    element.src = strValue;
+    element.style.backgroundImage = `url("${strValue}")`;
```

Comment:
any particular reason for background image instead of leaving as image with `object-fit: cover;`?

Reply:
@exterkamp points out a very good reason to keep them image elements :)